js/foundation.drilldown.js

```diff
@@ -395,6 +399,12 @@ Drilldown.defaults = {
    */
   backButton: '<li class="js-drilldown-back"><a tabindex="0">Back</a></li>',
   /**
+   * You can choose to position the back button either at the top or bottom of drilldown submenus.
+   * @option
+   * @example bottom
+   */
+  backButtonPosition: 'top',
```

Comment:
Comments should be descriptive (`This component does...`) or in the imperative forms if you talk about possibilities for the users (`Do that to make that...`).

So you can write instead :
```js
/**
 * Position the back button either at the top or bottom of drilldown submenus. The button is at the top by default.
  ...
 */
```

Also, please update the `backButton` documentation above. From `Prepended to submenu lists...` to something like `Prepended or appended (see backButtonPosition) to submenu lists...`.